collector/netclass_linux.go

```diff
@@ -31,6 +31,7 @@ import (
 
 var (
 	netclassIgnoredDevices = kingpin.Flag("collector.netclass.ignored-devices", "Regexp of net devices to ignore for netclass collector.").Default("^$").String()
+	netclassInvalidSpeed   = kingpin.Flag("collector.netclass.ignore-invalid-speed", "Ignore devices where the speed is invalid. This will be the default in 2.x.").Bool()
```

Comment:
```suggestion
	netclassInvalidSpeed   = kingpin.Flag("collector.netclass.ignore-invalid-speed", "Ignore devices where the speed is invalid. This will be the behaviour in 2.x.").Bool()
```

We don't plan to keep the flag in 2.x right?

Reply:
We could keep the flag, but make it hidden and change the default.